dora/core/client/fs/src/main/java/alluxio/client/file/DelegatingFileSystem.java

```diff
@@ -131,6 +132,12 @@ public URIStatus getStatus(AlluxioURI path, GetStatusPOptions options)
     return mDelegatedFileSystem.getStatus(path, options);
   }
 
+  @Override
+  public URIStatus getStatus(UfsUrl path, GetStatusPOptions options)
```

Comment:
where is `listStatus(UfsUrl...)`?

Reply:
In another [PR](https://github.com/Alluxio/alluxio/pull/17863)